package.json

```diff
@@ -18,11 +15,14 @@
   ],
   "repository": {
     "type": "git",
-    "url": "https://github.com/ember-cli/broccoli-builder"
+    "url": "https://github.com/ember-cli/broccoli"
```

Comment:
I'm wondering if we should be doing this stuff in https://github.com/ember-cli/broccoli instead 🤔